src/librustdoc/html/render/print_item.rs

```diff
@@ -186,6 +186,7 @@ pub(super) fn print_item(cx: &mut Context<'_>, item: &clean::Item, buf: &mut Buf
             }
         }
         clean::FunctionItem(..) | clean::ForeignFunctionItem(..) => "Function ",
+        clean::TestItem(..) => "Test ",
```

Comment:
`"Unit test "`

Reply:
We also want to call it on integration tests. If we change this to "Unit Test", it will also appear as Unit test in the integration tests.
With a command like `cargo rustdoc --test lib_test -- -Z unstable-options --document-tests --document-private-items`